doc/design/juju-api-design-specification.md

```diff
@@ -108,7 +108,7 @@ according paths care for those requests.
 
 The first one is the charm handler, which supports HTTP POST to add a local 
 charm to the store provider and HTTP GET to retrieve or list charm files. The
-second one is the tools handler, which supports HTTP POST for tje upload of
+second one is the tools handler, which supports HTTP POST for the upload of
```

Comment:
Doesn't this need 'the agent binaries handler, which supports ... the upload of agent binaries to the API server" ?